ovn/utilities/ovn-nbctl.c

```diff
@@ -2810,6 +2817,204 @@ normalize_prefix_str(const char *orig_prefix)
     }
 }
 
+
+static void
+nbctl_lr_policy_add(struct ctl_context *ctx)
+{
+    const struct nbrec_logical_router *lr;
+
+    char *error = lr_by_name_or_uuid(ctx, ctx->argv[1], true, &lr);
+    if (error) {
+        ctx->error = error;
+        return;
+    }
+
+    if (ctx->argc >= 5) {
+       VLOG_INFO("nbctl_lr_policy_add: priority %s match %s action %s",
+         ctx->argv[2], ctx->argv[3], ctx->argv[4]);
+    } else {
+       ctl_fatal("insufficient parameters");
+    }
+
+    int64_t priority = parse_priority(ctx->argv[2]);
```

Comment:
super nit:
Do we want this to be 64 bit int?

Reply:
Copied from ACL again :)